### DISCUSSION TITLE
How do I prevent storybook from opening a browser when it runs?

### DISCUSSION THREAD

**Discussion by remy** [storybookjs/storybook#12186] [Help] [answered]
I'm developing in Chrome but my default browser is Firefox, so storybook opens in Firefox - but I can't see anywhere on the website how to turn off this "feature" (I read the configure page, searched for "launch", "open" and "browser", it's also not in the faq - so if it's documented, it's well hidden!).

Thanks.

**Comment by shilman** [comment, marked as answer]
```
$ yarn storybook --help
Usage: start-storybook [options]
Options:
...
  --ci                          CI mode (skip interactive prompts, don't open browser)
```

**Reply by remy** [reply]
Thanks. (is that actually documented on the site? I did try to find it - obviously failed hard)

**Reply by jonniebigodes** [reply]
For future reference Storybook's Cli options are documented [here](https://storybook.js.org/docs/react/api/cli-options)

**Reply by waynevanson** [reply]
Use the flag --no-open instead of CI.

**Reply by brunobely** [reply]
IMO this should really be a prompt and not the default behavior. Feels a bit invasive to have a browser open automatically when all I was doing was initializing a project from the command line :(

edit: _especially_ invasive when telemetry is enabled by default

**Comment by camslice** [comment]
@remy you can also define the environment variable `BROWSER=none` as part of the node startup script as discovered here: https://github.com/storybookjs/storybook/issues/6201#issuecomment-812005603

`package.json` in the `scripts` section. Using the latest version of storybook.
```
"storybook": "BROWSER=none start-storybook",
```

**Comment by fireflysemantics** [comment]
I would like the browser to open, however it does not.  Any ideas?

https://stackoverflow.com/questions/76201084/opening-the-browser-automatically-when-running-storybook
